UnityProject/Assets/Scripts/Items/Weapons/Melee/StunBatonV2.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using AddressableReferences;
+using Mirror;
+using UnityEngine;
+
+[RequireComponent(typeof(Pickupable))]
+public class StunBatonV2 : NetworkBehaviour, ICheckedInteractable<HandActivate>, IServerSpawn
+{
+	public SpriteHandler spriteHandler;
```

Comment:
no need for this to be public. Make it private and get the component from the children.

Reply:
Fixed in fe4b08d. Set to private.